examples/gocean/eg6/main.f90

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Should this file be here? If so, please add license and document it.

Reply:
For now yes, till #647 is done - but it is indeed only temporary. Done, but I didn't put too much effort into details :)